modules/claims_api/app/controllers/claims_api/application_controller.rb

```diff
@@ -127,14 +127,12 @@ def veteran_from_headers(with_gender: false)
       vet.loa = if @current_user
                   @current_user.loa
                 else
-                  vet.loa = {
-                    current: header('X-VA-LOA').try(:to_i),
-                    highest: header('X-VA-LOA').try(:to_i)
-                  }
+                  { current: header('X-VA-LOA').try(:to_i), highest: header('X-VA-LOA').try(:to_i) }
                 end
       vet.mvi_record?
       vet.gender = header('X-VA-Gender') || vet.mvi.profile&.gender if with_gender
       vet.edipi = header('X-VA-EDIPI') || vet.mvi.profile&.edipi
+      vet.participant_id = header('X-VA-PID') || vet.mvi.profile&.participant_id
```

Comment:
Can we change this to try over & to be consistent with line 130? Would like to avoid mixing styles.

Reply:
Line 130 is stylistically different from 135 because it is both totally different in scope (this one is passing a single string value while the other is passing a hash) and rubocop method size prevents me from stylistically changing lines 133 through 135 to match the desired results, considering how much data would have to be passed to refactored methods keeping the same style of 133 and 134 made the most sense.